Bottom: 749b1dc762ccf6e89482ec3fabd1c2f020b1f3e4
Top:    e0e9789b2bd77176dfbe91eaeaf5d278fdd375db
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-21 16:50:43 +0100

Refresh of fixup-gtk-hid-drawing-routines

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index ac9674c..3354bc5 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1269,7 +1269,7 @@ poly_callback_clearing (const BoxType * b, void *cl)
 static int
 pour_callback_no_clear (const BoxType * b, void *cl)
 {
-  struct pin_info *i = (struct pin_info *) cl;
+  struct poly_info *i = (struct poly_info *) cl;
   PourType *pour = (PourType *)b;
 
   DrawPour (i->Layer, pour, 0);
@@ -1285,7 +1285,7 @@ pour_callback_no_clear (const BoxType * b, void *cl)
   static int
 pour_callback_clearing (const BoxType * b, void *cl)
 {
-  struct pin_info *i = (struct pin_info *) cl;
+  struct poly_info *i = (struct poly_info *) cl;
   PourType *pour = (PourType *)b;
 
   DrawPour (i->Layer, pour, 0);
